### PR TITLE
Make the favicon work universally

### PR DESCRIPTION
**Pull request by james7132**
The previous one only worked on traditional browsers. This enables it's use in a variety of platforms and use-cases. Also upped the compression on a lot of these.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -47,7 +55,6 @@
           <div id="navbar" class="collapse navbar-collapse" data-uib-collapse="navCollapsed">
             <ul class="nav navbar-nav">
               <li data-ng-class="{active: $state.includes('news')}"><a data-ui-sref="news.list">News</a></li>
-              <li data-ng-class="{active: $state.includes('projects')}"><a data-ui-sref="projects.list">Projects</a></li>
```
Comment (gscoppino) [inline src/index.html]: Why was this removed?

**Comment by james7132** [comment]
PTAL.

File: src/index.html [hunk 2]
```diff
@@ -12,7 +12,15 @@
 
     <base href="/">
 
-    <link rel="icon" href="img/favicon.ico" size="16x16 32x32 48x48 64x64" type="image/vnd.microsoft.icon">
+    <link rel="apple-touch-icon" sizes="180x180" href="/img/apple-touch-icon.png">
+    <link rel="icon" type="image/png" href="/img/favicon-32x32.png" sizes="32x32">
+    <link rel="icon" type="image/png" href="/img/favicon-16x16.png" sizes="16x16">
+    <link rel="manifest" href="/img/manifest.json">
```
Comment (gscoppino) [inline src/index.html]: Let's move `manifest.json` out of the `img` folder. I think its conventional to have it at the root next to things like `robots.txt`, and that would be better than in the `img` folder, anyway. While doing this, we'll need to modify the `build-other`/`watch-other` npm scripts to also copy over JSON files as well. New NPM scripts should be:

```
"build-other": "cpx \"src/**/*.{txt,json}\" dist --verbose"
"watch-other": "cpx \"src/**/*.{txt,json}\" dist --watch --verbose"
```

Only difference in the scripts is the brace expansion in the source glob.

Comment (james7132) [reply src/index.html]: Does it also make sense to do the same with browserconfig.xml?

Comment (gscoppino) [reply src/index.html]: Yeah, thanks for catching that.

**Comment by gscoppino** [comment]
I'm not too familiar with how favicons should work cross browser, but it appears that at the moment, Edge and Internet Explorer 11 are not using our favicon, in fact they are not even sending a HTTP request for it. It works fine in Chrome and Firefox.

EDIT: It appears IE and Edge are just being...IE and Edge. You need to open the site at least once in a standard session (not in private), in order for the favicon to load in these browser. After that, it presumably is cached, and it will load in private mode as well. KEK

**Review by gscoppino — APPROVED** [review]
It is working correct on all desktop browsers, and desktop / tile shortcuts for IE/Edge also are using the favicon correctly. I can't verify that it works right on every single device type (especially the iOS stuff), but I think its probably close enough to bring in.
